File: Python/Mahasiswa.py
# Import File:
from SivitasAkademik import SivitasAkademik
from Human import Human

# Kelas Mahssiwa (Child) - Human (Parent)
class Mahasiswa(SivitasAkademik):
    # Atribut Mahasiswa:
    __nim = ""
    __fakultas = ""
    __prodi = ""

    # Konstruktor
    def __init__(self, nik, nama, jenis_kelamin, asal_univ, fakultas, prodi, nim, email_edu):
        Human.__init__(self, nik, nama, jenis_kelamin)
        SivitasAkademik.__init__(self, asal_univ, email_edu)
        self.nim = nim
        self.fakultas = fakultas
        self.prodi = prodi

    # Setter
    def set_nim(self, nim):
        self.nim = nim

    def set_prodi(self, prodi):
        self.prodi = prodi

    def set_fakultas(self, fakultas):
        self.fakultas = fakultas

    # Getter
    def get_nim(self):
        return self.nim

    def get_prodi(self):
        return self.prodi

    def get_fakultas(self):
        return self.fakultas

File: PHP/Mahasiswa.php
<?php  

require("SivitasAkademik.php");

class Mahasiswa extends SivitasAkademik
{
    // Private attribute:
    private $nim = '';
    private $fakultas = '';
    private $prodi = '';
    private $gambar = '';

    // Konstruktor.
    public function __construct($nik = '', $nama = '', $jenis_kelamin = '', $asal_univ = '', $fakultas = '', $prodi = '', $nim = '', $email_edu = '', $gambar = '')
    {
        parent::__construct($nik, $nama, $jenis_kelamin, $asal_univ, $email_edu);
        $this->nim = $nim;        
        $this->fakultas = $fakultas;        
        $this->prodi = $prodi;
        $this->gambar = $gambar;
    }

    // Setter:
    // NIM.
    public function set_nim($nim = '')
    {
        $this->nim = $nim;
    }
    // Fakultas.
    public function set_fakultas($fakultas = '')
    {
        $this->fakultas = $fakultas;
    }
    // Program Studi.
    public function set_prodi($prodi = '')
    {
        $this->prodi = $prodi;
    }
    // Gambar
    public function set_gambar($gambar = '')
    {
        $this->gambar = $gambar;
    }

    // Getter:
    // NIM.
    public function get_nim()
    {
        return $this->nim;
    }
    // Fakultas.
    public function get_fakultas()
    {
        return $this->fakultas;
    }
    // Program Studi.
    public function get_prodi()
    {
        return $this->prodi;
    }
    // Gambar
    public function get_gambar()
    {
        return $this->gambar;
    }

    // Destruktor.
    public function __destruct()
    {
        
    }
}

?>